pkg/controller.v1/pytorch/pytorchjob_controller.go

```diff
@@ -372,7 +390,7 @@ func (r *PyTorchJobReconciler) UpdateJobStatus(job interface{},
 	for rtype, spec := range replicas {
 		status := jobStatus.ReplicaStatuses[rtype]
 		// Generate the label selector.
-		status.LabelSelector = metav1.FormatLabelSelector(r.GenLabelSelector(pytorchjob.Name, rtype))
+		status.Selector = metav1.FormatLabelSelector(r.GenLabelSelector(pytorchjob.Name, rtype))
```

Comment:
@tenzen-y I tested with this build. Strangely, Selector field is not seen to be populated and hence HPA is not working

Reply:
Hmmm...
I will check on my local.
Can you share your tested steps?

Reply:
Sorry. It was my mistake. I verified HPA with this build.

Reply:
I got it.